lib/testing/src/lib/process-services-cloud/pages/edit-task-filter-cloud-component.page.ts

```diff
@@ -78,6 +78,19 @@ export class EditTaskFilterCloudComponentPage {
         await browser.driver.sleep(1000);
     }
 
+    async expandFilter(): Promise<void> {
+        await this.isFilterDisplayed();
+        await BrowserActions.click(this.customiseFilter);
+        await this.checkCustomiseFilterHeaderIsExpanded();
+    }
+
+    async checkCustomiseFilterHeaderIsExpanded(): Promise<void> {
+        const expansionPanelExtended = element.all(by.css('mat-expansion-panel-header[class*="mat-expanded"]')).first();
```

Comment:
Could this `mat-expansion-panel-header[class*="mat-expanded"]` selector be replaced by `mat-expansion-panel-header.mat-expanded`?